source/renderer/app/components/voting/voting-registration-wizard-steps/widgets/VotingRegistrationDialog.scss

```diff
@@ -4,8 +4,8 @@
   text-align: center;
 
   :global {
-    .Dialog_title {
-      margin-bottom: 0;
+    .Dialog_contentWrapper::-webkit-scrollbar-track {
```

Comment:
@aleksandardjordjeviciohk shouldn't we put this into an override for the Dialog instead of the global here? 🤔

Reply:
This is obsolete now, as it was a leftover from previous implementation. Deleted now.